doc/manual/functions.rst

```diff
@@ -413,29 +413,25 @@ Optional Arguments
 ------------------
 
 In many cases, function arguments have sensible default values and therefore
-might not need to be passed explicitly in every call. For example, the
-library function :func:`parse(type,num,base) <parse>` interprets a string as a number
-in some base. The ``base`` argument defaults to ``10``. This behavior can be
+might not need to be passed explicitly in every call. For example, consider the
+function myfunc(x,y, z=1). The ``z`` argument defaults to ``1``. This behavior can be
```

Comment:
Use double backquotes `` around code blocks. Also, probably better call the function `f` for consistency with examples below.